.jenkins/pipelines/Azure/Windows/Jenkinsfile

```diff
@@ -58,15 +58,42 @@ def windowsLinuxElfBuild(String label, String version, String compiler, String b
     }
 }
 
+/**
+ * Compile open-enclave on Windows platform, generate NuGet package out of it, 
+ * install the generated NuGet package, and run samples tests against the installation.
+ */
 def windowsCrossCompile(String label, String build_type, String lvi_mitigation = 'None', String OE_SIMULATION = "0", String lvi_mitigation_skip_tests = 'OFF', List extra_cmake_args = []) {
     def node_label = AGENTS_LABELS["${label}-dcap"]
 
     stage("Windows ${label} ${build_type} with SGX LVI_MITIGATION=${lvi_mitigation}") {
         node(node_label) {
             withEnv(["OE_SIMULATION=${OE_SIMULATION}"]) {
                 timeout(GLOBAL_TIMEOUT_MINUTES) {
-                    // OFF value of quote provider until https://github.com/openenclave/openenclave-ci/pull/29 is merged
-                    oe.WinCompilePackageTest("build/X64-${build_type}", build_type, 'OFF', CTEST_TIMEOUT_SECONDS, lvi_mitigation, lvi_mitigation_skip_tests, extra_cmake_args)
+                    cleanWs()
```

Comment:
If you moved this here, you removed this from elsewhere, correct?

Reply:
This needs to be merged in first before it can be removed from the other location, yes.